src/accordion/accordion.ts

```diff
@@ -216,7 +247,17 @@ export class NgbAccordion implements AfterContentChecked {
    */
   @Output() panelChange = new EventEmitter<NgbPanelChangeEvent>();
 
-  constructor(config: NgbAccordionConfig) {
+  /**
+   * Event emitted right after the panel toggle happens, after the animation has been completed.
+   *
+   * See [NgbPanelCollapseEvent](#/components/accordion/api#NgbPanelCollapseEvent) for payload details.
+   */
+  @Output() ngbCollapseChange = new EventEmitter<NgbPanelCollapseEvent>();
```

Comment:
I think The API should be a bit different. We have 2 events everywhere else. Ex. `(shown)/(hidden)`. Could we maybe do this ?:

```html
<ngb-accordion (collapsed)="..." (expanded)="...">.  ← $event is the id of the panel here
  <ngb-panel id="first" (collapsed)="..." (expanded)="..."></ngb-panel> ← $event is undefined here
</ngb-accordion>
```

We've discussed this for navs and to me it's kinda similar reasoning.

`ngbCollapse` itself is different, because the event is in 2-way binding: `<div [(ngbCollapse)]="someBoolean"></div>`

@benouat would be nice to have your input on this too.

Reply:
I am 💯% with you here. We should have `(collapsed)/(expanded)`